src/rt-app_parse_config.c

```diff
@@ -526,27 +561,27 @@ parse_task_event_data(char *name, struct json_object *obj,
 	if (!strncmp(name, "timer", strlen("timer"))) {
 
 		tmp = get_string_value_from(obj, "ref", TRUE, "unknown");
-		if (!strncmp(tmp, "unique", strlen("unique")))
-				ref = create_unique_name(unique_name, sizeof(unique_name), tmp, tag);
-		else
-				ref = tmp;
 
-		i = get_resource_index(ref, rtapp_timer, opts);
+		if (!strncmp(tmp, "unique", strlen("unique"))) {
```

Comment:
Mmm. I'm wondering if for forking tasks timers should be unique by default?

Reply:
I don't think. I have in mind some UC where threads fork but still use the same timer

Reply:
ok